backend/gradle.properties

```diff
@@ -0,0 +1,9 @@
+# Properties to configure the gradle build
+
+exposed_version=0.25.1
```

Comment:
newest version is 0.28.1 on github, why version 25?

Reply:
good catch!